src/components/w2JobInfo.js

```diff
@@ -5,21 +5,20 @@ import { Link, useHistory } from 'react-router-dom'
 import { useDispatch, useSelector } from 'react-redux'
 
 import { LabeledInput } from './labeledInput'
-import { saveEmployerData } from '../redux/actions'
-import { getFormData } from '../redux/selectors'
+import { saveW2Data } from '../redux/actions'
 
 export default function W2JobInfo () {
   const { register, handleSubmit, errors } = useForm()
   const history = useHistory()
   // const variable dispatch to allow use inside function
   const dispatch = useDispatch()
 
-  const prevFormData = useSelector(state => getFormData(state, 'W2JobInfo'))
+  const prevFormData = useSelector(state => state.w2Info || {})
```

Comment:
Shouldn't this use the nullish coalescing operator (??) like the other components?